sale_product_multi_add/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016-2024 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Product Multi Add",
+    "summary": """
+        Sale Product Multi Add """,
+    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sale Management",
+    "version": "17.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["sale", "sale_management"],
```

Comment:
```suggestion
    "depends": ["sale_management"],
```
but I would go for not adding sale_management and left it as it was (because this module doesn't inherit/use anything from sale_management):
```suggestion
    "depends": ["sale"],
```